gimp/pictonode-gimp-plugin/GIMP 2.99/pictonode/window.py

```diff
@@ -62,6 +63,11 @@
 }
 """
 
+# define screen dimensions for window set up
```

Comment:
These should probably be fetched in the constructor, not in the global context, to avoid GDK bugs